js/tests/visual/carousel.html

```diff
@@ -53,6 +53,21 @@
 <script src="../vendor/jquery.min.js"></script>
 <script src="../../dist/util.js"></script>
 <script src="../../dist/carousel.js"></script>
+<script type="text/javascript">
```

Comment:
Please remove the `type="text/javascript"` for consistency with the rest of the project